### PR TITLE
React Capstone PR

### PR DESCRIPTION
**Pull request by DjibrilM**
## In the following pull request 
- styled the application
- integrated testing
- integrated APIs all
- created all required functionalities

### CODE REVIEW

**Comment by tugsem** [comment]
## Invalid Code Review Request ❌ 

Hi @DjibrilM , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

<h2>You have submitted a project, that doesn't have a proper readme. Please use the template and fill out all of the sections including live demo and video presentation link</h2>

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,179 @@
...
+- Add charts to display the statistics of each coin
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/olanikegloria/CryptoNike/issues).
```
Comment (Kossi-stack) [inline README.md]: - [ ] Kindly make sure the link you provoided is not broken

File: README.md [hunk 2]
```diff
@@ -1 +1,179 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Thanks to  [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio) and [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) for the design idea.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](LICENE) licensed.
```
Comment (Kossi-stack) [inline README.md]: - [ ] Kindly fix this link as well

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @DjibrilM,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly edit the PR's title to something more descriptive. it can be "React Capstone PR"

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (KanzaTahreem) [inline README.md]: - [ ] [As suggested](https://github.com/DjibrilM/react-capstone/pull/5#discussion_r1290777568), kindly make sure to fix the broken link 📌 Broken links in a README undermine user experience and hinder access to important information. Right now, it is redirecting to the following ⚠

image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @DjibrilM 

Great work on making the changes requested by a previous reviewer 👏! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the design
- ✅ Great job with CSS
- ✅ Descriptive README file
- ✅ Correct Linters

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (peterrobert) [reply README.md]: This change has been done well

File: README.md (hunk 2 above)
Comment (peterrobert) [reply README.md]: This change has been done well

File: README.md (hunk 1 above)
Comment (peterrobert) [reply README.md]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @DjibrilM, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**